### PR TITLE
Fix `delegate` default serialization with many args

### PR DESCRIPTION
**Pull request by masoud-msk**
Looks like a typo in default serialization which is `(...args) => JSON.stringify(...args)` instead of `(...args) => JSON.stringify(args)` as it doesn't work for example in case of the decorated method having more than 3 arguments or more than 2 arguments with the first one not being an object.

### CODE REVIEW

File: package-lock.json
Comment (vlio20) [inline package-lock.json:1]: please revert this change

File: src/delegate/delegatify.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export function delegatify<D = any, A extends any[] = any[]>(
   keyResolver?: (...args: A) => string,
 ): AsyncMethod<D, A> {
   const delegatedKeysMap = new Map<string, Promise<any>>();
-  const keyGenerator: (...args: any[]) => string = keyResolver ?? JSON.stringify;
+  const keyGenerator: (...args: any[]) => string = keyResolver ?? ((...args) => JSON.stringify(args));
```
Comment (vlio20) [inline src/delegate/delegatify.ts:8]: nice catch!

**Review by vlio20 — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (masoud-msk) [reply package-lock.json:1]: Done

**Comment by vlio20** [comment]
released in 2.0.9. Thanks @masoud-msk

**Pull request merged**